pkg/apis/componentconfig/edgecore/v1alpha1/types.go

```diff
@@ -399,8 +399,12 @@ type MetaManager struct {
 }
 
 type MetaServer struct {
-	Enable bool   `json:"enable"`
-	Server string `json:"server"`
+	Enable            bool   `json:"enable"`
+	AutonomySwitch    bool   `json:"autonomyswitch"`
```

Comment:
add comment fot this, and Explain what effect it would have if turned on

Reply:
`json:"autonomyswitch` -> `json:"autonomySwitch`

Reply:
```suggestion
	AutonomyWithoutAuthorization    bool   `json:"autonomyWithoutAuthorization"`
```
Will this name be better?

Reply:
Let's take this field name `autonomyWithoutAuthorization`.